sql/pre_install/tables.sql

```diff
@@ -195,8 +195,14 @@ CREATE TABLE IF NOT EXISTS _timescaledb_config.bgw_job (
     job_type            NAME        NOT NULL,
     schedule_interval   INTERVAL    NOT NULL,
     max_runtime         INTERVAL    NOT NULL,
-    max_retries         INT         NOT NULL,
+    max_retries         INTEGER     NOT NULL,
     retry_period        INTERVAL    NOT NULL,
+    proc_name           NAME        NOT NULL DEFAULT '',
+    proc_schema         NAME        NOT NULL DEFAULT '',
+    owner               NAME        NOT NULL DEFAULT CURRENT_ROLE,
```

Comment:
Why `NOT NULL` when you have the empty string below. Doesn't it make more sense to allow the field to be null instead of using the empty string to denote "undefined owner".

Reply:
this is only temporary to make the transition easier the last PR that removes job_type will remove these defaults